server/runtime/src/test/resources/scripts/stream_serverTask.js

```diff
@@ -0,0 +1,30 @@
+// mode=local,language=javascript
```

Comment:
I don't see where this script is invoked? I didn't think serializable stuff worked in js scripts?

Reply:
I carried over all the scripts from the old testsuite. I have an idea on how to fix this in 10.1.

Reply:
That would be pretty neat! 💯